tests/test_helpers/dom_helper.ts

```diff
@@ -189,17 +204,19 @@ export function setCheckboxValueAndTrigger(
  *
  * This comment is meant to leave a trace of this change in case some issues were to arise again.
  */
-export async function keyDown(key: string, options: any = {}): Promise<void> {
-  document.activeElement!.dispatchEvent(
-    new KeyboardEvent("keydown", Object.assign({ key, bubbles: true, cancelable: true }, options))
-  );
+export async function keyDown(
+  selector: string | EventTarget,
```

Comment:
wasn't it more close to really before ? By always dispatching on the activeElement ?